irc.md

```diff
@@ -34,6 +34,10 @@ You don't.
 
 You don't.
 
+**How do I get on discord?**
+
+Show up to a meeting and see if we let you in.
```

Comment:
I would actually link the HA discord, and specify that the private AHA channel is what's restricted.